src/name.rs

```diff
@@ -539,10 +539,17 @@ impl NamespaceResolver {
             .unwrap_or_else(|| Self::maybe_unknown(prefix))
     }
 
+    const XML_NAMESPACE: &'static [u8] = b"http://www.w3.org/XML/1998/namespace";
+    const XMLNS_NAMESPACE: &'static [u8] = b"http://www.w3.org/2000/xmlns/";
```

Comment:
I would like to see documentation on one of this consts about special processing of those prefixes with the link to the specification (https://www.w3.org/TR/xml-names11/#xmlReserved). Maybe just an adopted copy fro that specification.

Also, can we change type of these consts to `Namespace<'static>`?

I also prefer to have consts declared at the start of `impl` block, or you can declare them outside of `impl` block.

Reply:
I documented the reserved namespaces and changed their types. I also moved them to lazily initalized static outside of the impl. I did this as I wanted to have a HashMap that mapped the prefix to the Namespace. I also wanted to define the prefix and appropriate namespace in the same part of the code. I ended up using the once_cell, which is due to be incorporated in core at some point (https://doc.rust-lang.org/core/cell/struct.OnceCell.html#method.eq). The core support for OnceCell is currently experimental, so I hope this (using the crate) is acceptable until it's stabilized as I think it's a much better organization for the code.

Reply:
https://doc.rust-lang.org/core/cell/struct.LazyCell.html is the location of the Lazy related stuff in core.